src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/SunPKCS11.java

```diff
@@ -539,12 +549,51 @@ public long importKey(long hSession, CK_ATTRIBUTE[] attributes) throws PKCS11Exc
             if (attr.type == CKA_KEY_TYPE) {
                 keyType = attr.getLong();
             }
-            if (attr.type == CKA_VALUE) {
-                keyBytes = attr.getByteArray();
```

Comment:
Why is this no longer appropriate?

Reply:
That one is used for secret key, we need something else for rsa & ec keys.